config/initializers/strong_migrations.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+# Mark existing migrations as safe
+StrongMigrations.start_after = 20_210_622_170_835
```

Comment:
`20_210_622_170_835`? 😂 
`2021_06_22_17_08_35` is what this really comes from, right? I think. Does it really force you to specify the datetime as an integer?

Reply:
It does, and rubocop changes it to that format.

Reply:
It uses the database version as a cause for comparison:
`
  def self.check_enabled?(check, version: nil)
    if enabled_checks[check]
      start_after = enabled_checks[check][:start_after] || StrongMigrations.start_after
      !version || version > start_after
    else
      false
    end
  end
`